### PR TITLE
Actu twit

### PR DESCRIPTION
**Pull request by ThaiBangHOANG**

### CODE REVIEW

**Review by GeorgesFORTRY — DISMISSED** [review]
ok

File: src/App.jsx [hunk 1]
```diff
@@ -7,11 +8,28 @@ import NavBar from './components/NavBar/NavBar';
...
+      <HdfTwitter />
+      <TwitterTweetEmbed />
+      <TwitterTimelineEmbed />
+      <ContactForm />
+      <Footer />
+    </div>
+  );
+}
+
 dotenv.config();
```
Comment (LoicBrassart) [inline src/App.jsx]: Useless here

File: src/App.jsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { TwitterTweetEmbed, TwitterTimelineEmbed } from 'react-twitter-embed';
 import { useEffect, useState } from 'react';
 import axios from 'axios';
 import dotenv from 'dotenv';
```
Comment (LoicBrassart) [inline src/App.jsx:4]: Useless imports

Comment (LoicBrassart) [reply src/App.jsx:4]: And that makes me think: How did Prettier/ESlint let it go ?

**Review by LoicBrassart — APPROVED** [review]
~~SnE!~~
Yes, evils! My bad, checked the wrong case!

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Things to check out

File: src/App.jsx (hunk 2 above)
Comment (GeorgesFORTRY) [reply src/App.jsx:4]: We need it for the management of the order of the sections, seen in a previous PR

File: src/App.jsx (hunk 1 above)
Comment (GeorgesFORTRY) [reply src/App.jsx]: We need it for the management of the order of the sections, seen in a previous PR

Comment (LoicBrassart) [reply src/App.jsx]: What I mean is, check the whole file: there are **two** App components

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
BiHeaded App component iz bad component

**Comment by GeorgesFORTRY** [comment]
I haven't seen that 😱

**Review by GeorgesFORTRY — APPROVED** [review]
ok

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**